projects/storefrontapp-e2e-cypress/cypress/integration/regression/reset-password/forgot-password.e2e-spec.ts

```diff
@@ -0,0 +1,23 @@
+context('Forgot Password Page', () => {
+  before(() => {
```

Comment:
before go to forgot-password page, we need clear session to make sure there is really no user token exist (no user login).
cy.window().then(win => win.sessionStorage.clear());

Reply:
Fixed.  Good idea.  I didn't know we could do that.